designs/cloudWatch/cloudWatchIntegration.md

```diff
@@ -0,0 +1,76 @@
+# CloudWatch Logs Experience
+
+* **Type**: UX Design
+* **Status**: Proposal
```

Comment:
Even with a proposal, avoid language like "would/will". Write as if the system already exists. The approved form of the document lives on to document the actual experience.

Example:
A new AWS Explorer root node would be added for the CloudWatch service.
to
The AWS Explorer gains a root node for the CloudWatch service.